src/org/apache/lucene/analysis/hunspell/Dictionary.m

```diff
@@ -1132,9 +1132,9 @@ void OrgApacheLuceneAnalysisHunspellDictionary_parseAffixWithJavaUtilTreeMap_wit
 }
 
 JavaNioCharsetCharsetDecoder *OrgApacheLuceneAnalysisHunspellDictionary_getJavaEncodingWithNSString_(OrgApacheLuceneAnalysisHunspellDictionary *self, NSString *encoding) {
-  if ([@"ISO8859-14" isEqual:encoding]) {
-    return create_OrgApacheLuceneAnalysisHunspellISO8859_14Decoder_init();
-  }
+//  if ([@"ISO8859-14" isEqual:encoding]) {
```

Comment:
I have improved the script on the [mobilelucene PR](https://github.com/lukhnos/mobilelucene/pull/2) to better post process the generated files.

The relevant commit is this one: https://github.com/lukhnos/objclucene/pull/2/commits/f83aa468f25d2bfadf1c9004319d9adb8aa80c4b

The code has been generated by using the latest version of the `translate.py` script.